MAINTAINERS.yml

```diff
@@ -1841,6 +1838,489 @@ West:
     labels:
         - "area: West"
 
+"West project: canopennode":
+  status: maintained
+  maintainers:
+    - henrikbrixandersen
+  files:
+    - modules/canopennode/
+  labels:
+    - manifest-canopennode
+    - "area: CAN"
+
+"West project: civetweb":
+  status: obsolete
+  files:
+    - samples/net/civetweb/
+    - modules/Kconfig.civetweb
+  labels:
+    - manifest-civetweb
+    - "area: Networking"
+    - "area: civetweb"
+
+"West project: cmsis":
+  status: odd fixes
+  collaborators:
+    - nashif
+  files:
+    - modules/Kconfig.cmsis
+    - modules/Kconfig.cmsis_dsp
+    - modules/Kconfig.cmsis_nn
+  labels:
+    - manifest-cmsis
+
+"West project: edtt":
+  status: odd fixes
+  collaborators:
+    - erbr-ot
+    - kruithofa
+    - thoh-ot
+    - MariuszSkamra
+  files: []
+  labels:
+    - manifest-edtt
+
+"West project: fatfs":
+  status: maintained
+  maintainers:
+    - de-nordic
+  files: []
+  labels:
+    - manifest-fatfs
+
+"West project: hal_altera":
+  status: obsolete
```

Comment:
Checking with @nashif about this one

Reply:
how was this determined? it is not obsolete.

Reply:
@nashif who is the collaborator then? Does it have any?

Reply:
you can put me.

Reply:
OK. https://github.com/zephyrproject-rtos/zephyr/pull/44614#discussion_r846299080 for details on how it was determined originally